vol_test_parallel.c

```diff
@@ -42,54 +42,54 @@ uint64_t vol_cap_flags;
+    X(VOL_TEST_GROUP, "group", vol_group_test_parallel, 1)                                                   \
+    X(VOL_TEST_DATASET, "dataset", vol_dataset_test_parallel, 1)                                             \
+    X(VOL_TEST_DATATYPE, "datatype", vol_datatype_test_parallel, 1)                                          \
+    X(VOL_TEST_ATTRIBUTE, "attribute", vol_attribute_test_parallel, 1)                                       \
+    X(VOL_TEST_LINK, "link", vol_link_test_parallel, 1)                                                      \
+    X(VOL_TEST_OBJECT, "object", vol_object_test_parallel, 1)                                                \
+    X(VOL_TEST_MISC, "misc", vol_misc_test_parallel, 1)                                                      \
+    X(VOL_TEST_ASYNC, "async", vol_async_test_parallel, 1)                                                   \
+    X(VOL_TEST_MAX, "", NULL, 0)
 #else
-#define VOL_PARALLEL_TESTS                                               \
-    X(VOL_TEST_NULL,      "",          NULL,                        0)   \
-    X(VOL_TEST_FILE,      "file",      vol_file_test_parallel,      1)   \
-    X(VOL_TEST_GROUP,     "group",     vol_group_test_parallel,     1)   \
-    X(VOL_TEST_DATASET,   "dataset",   vol_dataset_test_parallel,   1)   \
-    X(VOL_TEST_DATATYPE,  "datatype",  vol_datatype_test_parallel,  1)   \
-    X(VOL_TEST_ATTRIBUTE, "attribute", vol_attribute_test_parallel, 1)   \
-    X(VOL_TEST_LINK,      "link",      vol_link_test_parallel,      1)   \
-    X(VOL_TEST_OBJECT,    "object",    vol_object_test_parallel,    1)   \
-    X(VOL_TEST_MISC,      "misc",      vol_misc_test_parallel,      1)   \
-    X(VOL_TEST_MAX,       "",          NULL,                        0)
+#define VOL_PARALLEL_TESTS                                                                                   \
+    X(VOL_TEST_NULL, "", NULL, 0)                                                                            \
+    X(VOL_TEST_FILE, "file", vol_file_test_parallel, 1)                                                      \
+    X(VOL_TEST_GROUP, "group", vol_group_test_parallel, 1)                                                   \
+    X(VOL_TEST_DATASET, "dataset", vol_dataset_test_parallel, 1)                                             \
+    X(VOL_TEST_DATATYPE, "datatype", vol_datatype_test_parallel, 1)                                          \
+    X(VOL_TEST_ATTRIBUTE, "attribute", vol_attribute_test_parallel, 1)                                       \
+    X(VOL_TEST_LINK, "link", vol_link_test_parallel, 1)                                                      \
+    X(VOL_TEST_OBJECT, "object", vol_object_test_parallel, 1)                                                \
+    X(VOL_TEST_MISC, "misc", vol_misc_test_parallel, 1)                                                      \
+    X(VOL_TEST_MAX, "", NULL, 0)
```

Comment:
Not sure how important it would be to turn the formatter off here

Reply:
This one doesn't look too important. I'm guessing it's just trying to pad out spaces to match the line length, but we could maybe check clang format's options to see what it's trying to do here.